lib/texts.js

```diff
@@ -341,6 +341,10 @@ ${leaves}`,
 	'errorType.invalidPackage.appleTvDevicePackageOnSim': () => 'An Apple TV app package cannot be launched on simulator device.',
 	'errorType.configuratorError': () => 'Make sure that Apple Configurator 2 and Automation Tools are installed. Please see our docs.',
 	'errorType.appStoreBuild': () => 'Can’t install App Store distribution build.',
+	'errorType.notSupportedDriver': () => 'Screenshots not supported by the device driver.',
```

Comment:
Wrong messages. Correct one are defined in the ticket description, not in comments. Please, fix.